Wed, 08 Sep 2021 10:50:50-run_all_case.py-Register_case.py-[line:85]-INFO-[日志信息]: 开始执行脚本test_Wrong_Num：

